Userland/Libraries/LibWeb/HTML/Window.h

```diff
@@ -313,6 +327,18 @@ class Window final
     // https://html.spec.whatwg.org/multipage/nav-history-apis.html#dom-window-status
     // When the Window object is created, the attribute must be set to the empty string. It does not do anything else.
     String m_status;
+
+    // https://www.w3.org/TR/event-timing/#sec-modifications-HTML
+    Vector<JS::NonnullGCPtr<EventTiming::PerformanceEventTiming>> m_entries_to_be_queued;
+    JS::GCPtr<EventTiming::PerformanceEventTiming> m_pending_first_pointer_down { nullptr };
```

Comment:
No need for `{ nullptr }` on a `GCPtr` -- they are always default-initialized to null.

Reply:
Thanks for the tips - updated